studio/components/interfaces/Billing/PlanSelection/Plans/PlanCTAButton.tsx

```diff
@@ -20,13 +20,23 @@ const PlanCTAButton = ({ plan, currentPlan, onSelectPlan }: PlanCTAButtonProps)
   const isTeamTier = currentPlan?.supabase_prod_id === PRICING_TIER_PRODUCT_IDS.TEAM
 
   const getButtonType = (plan: any, currentPlan: any) => {
+    if (
+      currentPlan.supabase_prod_id !== PRICING_TIER_PRODUCT_IDS.ENTERPRISE &&
+      plan.name === 'Enterprise'
+    ) {
+      return 'default'
+    }
+
     if (['Free tier', 'Free plan'].includes(plan.name)) {
       // Free is always default
       return 'default'
     } else if (currentPlan.prod_id === STRIPE_PRODUCT_IDS.FREE) {
       // If the current plan is free, other plans are primary
       return 'primary'
-    } else if (currentPlan.prod_id === STRIPE_PRODUCT_IDS.TEAM && !['Team tier', 'Team plan'].includes(plan.name)) {
+    } else if (
+      currentPlan.prod_id === STRIPE_PRODUCT_IDS.TEAM &&
+      !['Team tier', 'Team plan'].includes(plan.name)
```

Comment:
you already know what the issue is i'm commenting here about :p

Reply:
Component will be deleted in the next 2 weeks (everything under Billing/* which is not in V2)

Team tier was just backwards comp, as it checked the Stripe name, could be removed, but component will get deleted soon anyway